docker/spring-cloud-contract-docker/project/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
```

Comment:
Let's leave the previous Gradle version

Reply:
Wasn't even intentional, tbh. I'll change it.